backend/src/main/resources/application-dev.yml

```
@@ -0,0 +1,18 @@
+# Application profile dev
+
+spring:
+  jpa:
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+
+logging:
+  level:
+    org.hibernate.SQL: DEBUG # SQL logs uniquement en dev
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)